### DISCUSSION TITLE
Filter "find feature neighbors" with periodic boundary condition

### DISCUSSION THREAD

**Discussion by vivelakorea** [BlueQuartzSoftware/DREAM3D#1030] [Q&A]
Hi,

I've got dream3d-reconstructing pipeline, which uses grainID_n.txt and orientations_n.txt as input (attatches below)

In the pipeline, there is a "find feature neighbors" filter, but I found out that this does not find neighboring grain across the boundary.Capture.PNG

![Capture](https://github.com/BlueQuartzSoftware/DREAM3D/assets/40027828/9e11b956-7c64-4308-805b-5ece22892089)

For example, as in capture above, when the grain 36 and grain 2 are neighboring across the boundary, the "find feature neighbors" cannot find the neighbor.

How may I modify this?

Best regards,
Gyu-Jang Sim

**Comment by StopkaKris** [comment]
The author also asked this question on the DREAM.3D help forum at the link below. A solution was provided, so the thread here may be closed.

https://groups.google.com/g/dream3d-users/c/_yjFGOdAdBA/m/gpCK-LopAQAJ
